src/data/skill-desc.csv

```diff
@@ -51,9 +51,9 @@
 "レース終盤に前方が詰まると\nお告げの力で道を切り開く","<size=22>When the front is packed during the last leg of the race, the oracle's power will open a path</size>"
 "レース終盤で3番手の時に負けそうにな\nると秘めた闘志に火が付き速度が上がる","<size=22>When you're in 3rd place at the end of the race and about to lose, your speed increases</size>"
 "レースを冷静に運ぶことができると残り\n200mで実力発揮、速度がすごく上がる","<size=22>After proceeding through the race calmly, your speed drastically increases during the last 200m</size>"
-"中盤で2回スキルを使うと\nよりリラックスして持久力が回復する",""
+"中盤で2回スキルを使うと\nよりリラックスして持久力が回復する","<size=22>Restore endurance if you activate two skills in the middle part of the race</size>"
 "最終直線で前の方で追いすがっていると\n不屈の闘志で速度が上がる","<size=22>Increase speed in the last straight when chasing someone close to the lead</size>"
-"スキルで持久力を回復すると\nレース後半で前のほうにいる時に\nレッツラゴー！と速度を上げる",""
+"スキルで持久力を回復すると\nレース後半で前のほうにいる時に\nレッツラゴー！と速度を上げる","<size=22>In the second half of the race, increase speed when you recover endurance with your skills while being in front</size>"
```

Comment:
```suggestion
"スキルで持久力を回復すると\nレース後半で前のほうにいる時に\nレッツラゴー！と速度を上げる","<size=22>In race's second half, increase speed when recovering endurance with skills while in front</size>"
```

Reply:
`In the race second half` or even `In race second half` sounds even shorter and still correct to me

Reply:
Dropping the "the" makes it sound awkward, i'd personally most prefer "In the race's second half" as that sounds most correct and only adds two characters to your first suggestion here, while still being shorter than the original.

Reply:
`In race second half, increase speed when…` actually does not sound awkward to me at all; it actually sounds better than the original longer one if I ignore the length; need a third opinion here

Reply:
I'm good with waiting for a third opinion